git-link.el

```diff
@@ -819,6 +822,22 @@ shown via annotate in bitbucket."
         "annotate"
       "src")))
 
+;;;###autoload
+(defun git-link-diffrent-branch (branch)
+  "Invoke `git-link', but with the `branch' name set to a different
+branch than the one you're currently working on."
+  (interactive "P")
+  (let* ((default-remote-branch-name "main")
```

Comment:
The thing to consider here is this `main` usage.

Some people might have `master` or something else.